app/assets/locales/locale-de.json

```diff
@@ -1,371 +1,371 @@
+            "memo_missing": "Sie haben nicht die Kontrolle über den Notitzschlüssel. Damit Sie die Notitzfunktion benutzen können, muss unter Einstellungen -> Konten -> Berechtigungen, ein von ihnen kontrollierter Schlüssel ausgewählt werden.",
             "not_found": "Das Konto %(name)s existiert nicht. Bitte prüfen Sie die Schreibweise!",
             "not_yours": "Nicht Ihr Konto",
             "unknown": "Unbekannter Kontoname"
         },
-        "estimate_value": "Estimated Account value",
+        "estimate_value": "Geschätzter Kontowert",
         "existing_accounts": "Existierendes Konto",
-        "existing_user": "Existing user",
+        "existing_user": "Existierender Benutzer",
         "filter_orders": "Filter",
         "follow": "Kontakt hinzufügen",
-        "hidden_accounts_row": "Hidden Accounts",
-        "hide_hidden": "Active",
-        "hide_ignored": "Hide hidden accounts",
-        "history": "Historie",
-        "history_error": "Failed to download the account history, please try again",
-        "home": "Home",
+        "hidden_accounts_row": "Versteckte Konten",
+        "hide_hidden": "Aktiv",
+        "hide_ignored": "Blende versteckte Konten aus",
+        "history": "Verlauf",
+        "history_error": "Verlaufsfehler",
+        "home": "Zuhause",
         "hour_24": "24h-Wechsel",
         "hour_24_short": "24h",
-        "id": "Konto ID",
+        "id": "Konto Identifikation",
         "identicon": "Identicon",
-        "ignore": "Hide",
-        "in_open": "This is the amount of %(asset)s you have in open orders.",
-        "in_open_value": "This is the estimated value of the %(asset)s you have in open orders.",
-        "intro_text_1": "Your Decentralized Exchange",
-        "intro_text_2": "<b>The DEX</b> is inherently decentralized, enabling you to trade the BitShares core token (BTS) and a range of trustless price-stable, market-pegged assets such as bitUSD, bitCNY, bitBTC, bitGold and more. These assets can all be traded with <b>zero</b> counter-party risk, putting you in total control of your funds.",
-        "intro_text_3": "<b>The DEX</b> also enables creation of user-issued assets (UIAs), which can be traded against any other UIAs or trustless assets on the platform.  One well-known issuer on The DEX is <b>OpenLedger</b>, offering a host of assets including BTC, ETH, STEEM, DASH, LTC, DOGE, and many more",
-        "intro_text_4": "This gives cryptocurrency traders a unique blend of traditional and decentralized trading, all in a familiar exchange interface. Happy trading!",
-        "intro_text_title": "Welcome to BitShares",
-        "login_with": "Login with",
-        "market_actions": "Market actions",
+        "ignore": "Verstecke",
+        "in_open": "Der Betrag von %(asset)s, steckt in Ihren offenen Aufträgen.",
+        "in_open_value": "Der geschätze Wert von %(asset)s, steckt in Ihren offenen Aufträgen.",
```

Comment:
Was tut das Komma in `in_open_value` und `in_open`?

Reply:
Komma wird entfernt.